README.md

```diff
@@ -26,6 +28,12 @@ Examples:
     console.log(randomWords({ exactly: 2 }));
     ['beside', 'between']
 
+    console.log(randomWords({ min: 2, max: 3, seed: 'my-seed' }));
+    console.log(randomWords({ min: 2, max: 3, seed: 'my-seed' }));
```

Comment:
I still would like to see you explain the reasoning here. Something like, "// This call to `randomWords()` will yield exactly the same result as the previous call because the `seed` phrase is identical." in front of the second `console.log` and move the results of the first `console.log()` up underneath that `console.log`. Sorry to be so picky, I just think this would be a lot clearer.

Reply:
No problem! Updated 👍